src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/PetServiceAccountProvider.scala

```diff
@@ -6,9 +6,7 @@ import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 
 import scala.concurrent.Future
 
-/**
-  * Created by rtitle on 12/5/17.
-  */
+@deprecated(message = "Use PetsPerProjectServiceAccountProvider", since = "")
```

Comment:
is there a need to keep this around? can we just nuke it?

Reply:
We can probably nuke it, esp if the old Sam code is being nuked.